packages/components/buttons/icon-button/src/icon-button.story.js

```diff
@@ -34,8 +34,8 @@ storiesOf('Components|Buttons', module)
             icon={createElement(icons[select('icon', iconNames, iconNames[0])])}
             onClick={action('onClick')}
             label={text('label', 'Accessibility text')}
-            isToggleButton={boolean('isToggleButton', true)}
-            isToggled={boolean('isToggled', true)}
+            isToggleButton={boolean('isToggleButton', false)}
+            isToggled={boolean('isToggled', false)}
```

Comment:
I felt like this 2 options shouldn't be active by default, since a toggle-button seems to be less frequently needed than a regular button.

Reply:
+1 to that.